locations/spiders/footlocker.py

```diff
@@ -1,109 +1,46 @@
-        addr_full = response.xpath(
-            '//meta[@itemprop="streetAddress"]/@content'
-        ).extract_first()
-        city = response.xpath(
-            '//meta[@itemprop="addressLocality"]/@content'
-        ).extract_first()
-        state = response.xpath('//*[@itemprop="addressRegion"]//text()').extract_first()
-        postcode = response.xpath('//*[@itemprop="postalCode"]/text()').extract_first()
-        country = response.xpath(
-            '//*[@itemprop="addressCountry"]/text()'
-        ).extract_first()
-
-        # Foot Locker provides lat/lon via http://schema.org/GeoCoordinates microdata.
-        lat = float(
-            response.xpath('//meta[@itemprop="latitude"]/@content').extract_first()
-        )
-        lon = float(
-            response.xpath('//meta[@itemprop="longitude"]/@content').extract_first()
-        )
-
-        properties = {
-            "ref": ref,
-            "name": name,
-            "addr_full": addr_full,
-            "city": city,
-            "state": state,
-            "postcode": postcode,
-            "country": country,
-            "phone": phone,
-            "website": website,
-            "lat": lat,
-            "lon": lon,
-        }
-
-        # Hours are listed in OpenStreetMap's opening_hours format but per day, so we need to convert the list of hours
-        # into a single string using the OpeningHours object.
-        hours = response.xpath('//*[@itemprop="openingHours"]/@content').extract()
-        parsed_hours = self.parse_hours(hours)
-        if parsed_hours.as_opening_hours():
-            properties["opening_hours"] = parsed_hours.as_opening_hours()
-
-        yield GeojsonPointItem(**properties)
+    sitemap_urls = [
+        "https://stores.footlocker.com/sitemap.xml",
+        "https://stores.footlocker.at/sitemap.xml",
+        "https://stores.footlocker.be/sitemap.xml",
+        "https://stores.footlocker.ca/sitemap.xml",
+        "https://stores.footlocker.cz/sitemap.xml",
+        "https://stores.footlocker.de/sitemap.xml",
+        "https://stores.footlocker.dk/sitemap.xml",
+        "https://stores.footlocker.es/sitemap.xml",
+        "https://stores.footlocker.fr/sitemap.xml",
+        "https://stores.footlocker.gr/sitemap.xml",
+        "https://stores.footlocker.hu/sitemap.xml",
+        "https://stores.footlocker.ie/sitemap.xml",
+        "https://stores.footlocker.it/sitemap.xml",
+        "https://stores.footlocker.lu/sitemap.xml",
+        "https://stores.footlocker.nl/sitemap.xml",
+        "https://stores.footlocker.no/sitemap.xml",
+        "https://stores.footlocker.co.nz/sitemap.xml",
+        "https://stores.footlocker.pl/sitemap.xml",
+        "https://stores.footlocker.pt/sitemap.xml",
+        "https://stores.footlocker.se/sitemap.xml",
+        "https://stores.footlocker.co.uk/sitemap.xml",
+    ]
+    sitemap_rules = [
+        ("/en/", "parse"),
```

Comment:
Does this list need to include all the domains in the sitemap list?

Reply:
The "sitemap_urls" list is comprehensive as far as I can see.
The "sitemmap_rules" list is saying for the English speaking countries use whatever pages they produce. For non English speaking countries they publsih both an English version and "local". We take the English languag page rather than the local language so as not to duplicate.

Reply:
Ah that makes sense, thanks for the explanation!